ui/src/app/pages/admin/admin-workspace.tsx

```diff
@@ -168,8 +173,10 @@ const NameCell = (props: NameCellProps) => {
 
   // remove first check after RW-5626
   const isNotebook =
-    NOTEBOOKS_DIRECTORY === parseLocation(file, bucket) &&
-    filename.endsWith(NOTEBOOKS_SUFFIX);
+    (NOTEBOOKS_DIRECTORY === parseLocation(file, bucket) &&
```

Comment:
This condition seem to say thatJupyter notebooks need to be in the notebooks directory, but R and Rmd files do not. Is that what we want?